code/modules/clothing/head/misc_special.dm

```diff
@@ -372,6 +372,12 @@
 	item_state = "rflower"
 	w_class = WEIGHT_CLASS_TINY
 
+/obj/item/clothing/head/custodianhelm
+	name = "Custodian Helmet"
```

Comment:
lowercase all of this; if Custodian is a proper noun then prefix it with \improper ie "\improper Gygax chassis"